remappers/HumanDynamicsRemapper/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+yarp_prepare_plugin(human_dynamics_remapper
+    TYPE hde::devices::HumanDynamicsRemapper
+    INCLUDE HumanDynamicsRemapper.h
+    CATEGORY device
+    ADVANCED
+    DEFAULT ON)
+
+yarp_add_plugin(HumanDynamicsRemapper
+    HumanDynamicsRemapper.cpp
+    HumanDynamicsRemapper.h)
+
+target_include_directories(HumanDynamicsRemapper PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+
+target_link_libraries(HumanDynamicsRemapper PUBLIC
+    IHumanDynamics
+    HumanDynamicsMsg
+    YARP::YARP_OS
+    YARP::YARP_dev
+    YARP::YARP_init)
+
+yarp_install(
+    TARGETS HumanDynamicsRemapper
+    COMPONENT runtime
+    LIBRARY DESTINATION ${YARP_DYNAMIC_PLUGINS_INSTALL_DIR}
+    ARCHIVE DESTINATION ${YARP_STATIC_PLUGINS_INSTALL_DIR}
+    YARP_INI DESTINATION ${YARP_PLUGIN_MANIFESTS_INSTALL_DIR})
```

Comment:
Please remember to put a blank line a the end of files (https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log).